library/std/src/sys/itron/abi.rs

```diff
@@ -0,0 +1,176 @@
+//! ABI for μITRON derivatives
+pub type int_t = crate::os::raw::c_int;
+pub type uint_t = crate::os::raw::c_uint;
+pub type bool_t = int_t;
+
+/// Kernel object ID
+pub type ID = int_t;
+
+/// The special value of `ID` representing the current task.
+pub const TSK_SELF: PRI = 0;
+
+/// Relative time
+pub type RELTIM = u32;
+
+/// Timeout (relative time or infinity)
+pub type TMO = u32;
+
+pub type HRTTIM = u32;
+
+/// The maximum valid value of `RELTIM`
+pub const TMAX_RELTIM: RELTIM = 4_000_000_000;
+
+/// System time
+pub type SYSTIM = u64;
+
+/// Error code type
+pub type ER = int_t;
+
+/// Error code type, `ID` on success
+pub type ER_ID = int_t;
+
+/// Task or interrupt priority
+pub type PRI = int_t;
+
+/// The special value of `PRI` representing the current task's priority.
+pub const TPRI_SELF: PRI = 0;
+
+/// Object attributes
+pub type ATR = uint_t;
+
+/// Sort waiters by priority
+pub const TA_TPRI: ATR = 0x01;
+
+/// Use the priority inheritance protocol
+#[cfg(target_os = "solid-asp3")]
+pub const TA_INHERIT: ATR = 0x02;
+
+/// Activate the task on creation
+pub const TA_ACT: ATR = 0x01;
+
+pub type FLGPTN = uint_t;
```

Comment:
missing comment?

Reply:
Addressed in ef5a4088b6b1e9b2bec9bcfc9539c0017ba43da5